docs/tutorial_tx_service_set_delegate.md

```diff
@@ -0,0 +1,91 @@
+---
+id: tutorial_tx_service_set_delegate
+title: Set a delegate on the Safe transaction service
```

Comment:
Is it missing information on how to actually submit a transaction as a delegate or will this be a separate tutorial?

Reply:
Tx submission is the same as if it would be an owner. I'll mention that and link to the other tutorial.